package.json

```diff
@@ -23,7 +23,6 @@
     "e2e": "cypress open",
     "build:packages": "npm run build -w=packages/remix-runtime -w=packages/remix-adapter -w=packages/remix-edge-adapter",
     "build:packages:watch": "npm run build:watch -w=packages/remix-runtime & npm run build:watch -w=packages/remix-edge-adapter",
-    "build:edge-demo": "npm run build -w=packages/edge-demo-site",
```

Comment:
We don't need this custom script anymore. Just use `npm run build -w=packages/edge-demo-site` in the netlify.toml now.